tests/test_initial_point.py

```diff
@@ -152,6 +152,34 @@ def test_adds_jitter(self):
         assert fn(0) == fn(0)
         assert fn(0) != fn(1)
 
+    def test_jitter_scale(self):
```

Comment:
Can you simplify this test? If you test the default 1.0 jitter and a very large amount like 1000.0 a single draw should be enough to test the change.

Reply:
The large jitter should be >> 10 in magnitude, with p=1-10/1000, whereas with default jitter should be smaller than 1, with p=1